BookingProject/BookingWebAPI/Migrator/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Migrator.Interfaces;
+using Migrator.Services;
+using BookingWebApi.Infrastructure.Data;
+using BookingWebApi.Domain.Entities;
+using BookingWebApi.Application.Decorators;
+using AutoMapper;
+using BookingWebApi.Application.Interfaces;
+
+namespace Migrator
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+
+            var migrationService = host.Services.GetRequiredService<IDataMigrationService>();
+
+            var result = await migrationService.MigrateData("Company.json");
```

Comment:
Accept it from `string[] args`, find parameter for example `--file`. This will be more convenient to use your tool

Reply:
Fixed